### ISSUE TITLE
Rewrite and migrate DIC rates pages into Drupal

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#3040]
**Goal**
Legacy DIC rates pages have been rewritten in heroku but have not been shared with stakeholders. Due to table anomolies, these pages also need to be rewritten for Drupal components. DIC rates links are currently being moved to the Disability hub. AFter that happens, the rewritten new pages need to replace them.

**Source legacy URLs:**
https://www.benefits.va.gov/Compensation/current_rates_dic.asp
https://www.benefits.va.gov/Pension/current_rates_Parents_DIC_pen.asp

**New URLs:**
@mnorthuis needs to assign/determine exact URL paths and nav locations.
/[NEWHUB-NOT-BURIALS]/survivor-dic-rates/  
/[NEWHUB-NOT-BURIALS]/parent-dic-rates/

**Acceptance criteria**
[ ] Rewrite/revise DIC rates pages
[ ] Migrate into Drupal (manually)
[ ] Coordinate with FE engineer and Mikki to get redirects of legacy URLs ready on go-live. 

@RLHecht  - you're already working on this, so assigning you here. Just pulling this part out of the 'move DIC from Burials to Disability' ticket, so that part can move forward without having to wait for these 2 pages going live at the same time, per discussion. 

@mnorthuis  ^ FYSA - the prior new urls that were created for these 2 rates pages used burials-memorial in the path, so these will need your new look post move to Disability. 

**Acceptance Criteria:**
- Re-write content
- Coordinate redirects (separate ticket)

**Comment by RLHecht** [comment]
@jenniferlee-dsva I reworked the parent DIC rates page again. I added the "Note" text and the sentence above to the information above each group of accordions. I know it is repetitive but it seemed like a better way to keep the directions and info close to the tables. I also took off the Historic Rates and Title 38 Regulations line from the anchor links. I still need to have someone else check the numbers to make sure they were copied over correctly (that was the mistake with why the monthly payment wasn't adding up correctly).
http://preview-prod.vfs.va.gov/preview?nodeID=1013

**Comment by RLHecht** [comment]
@jenniferlee-dsva I tweaked the DIC surviving spouse and child benefit rates page as well. Let me know what you think and then I can have @AnneHurley or @lunascoop check the numbers on these tables as well. 
preview-prod.vfs.va.gov/preview?nodeId=1012

**Comment by RLHecht** [comment]
@jenniferlee-dsva FYI Lou checked the tables on both of these pages against the source content. A few edits needed to be made and numbers now all match what was originally provided.

**Comment by jenniferlee-dsva** [comment]
@RLHecht - I'll take a look at the copy updates this week (the edits you made last week), and will reach out w/any questions. 

@Karak888  -- Could you create a subtask/subticket to track the remaining part of this work? 
[ ] Coordinate with FE engineer and Mikki to get redirects of legacy URLs ready on go-live

**Comment by jenniferlee-dsva** [comment]
@RLHecht - The CMS is down and I can't access the preview, so I'll keep trying tomorrow.

**Comment by jenniferlee-dsva** [comment]
I am still not able to see the preview page ^ @RLHecht.  Going to Slack CMS team.

**Comment by jenniferlee-dsva** [comment]
@RLHecht - I reviewed and made some updates to the surviving parents page: https://prod.cms.va.gov/2019-va-dic-surviving-parent-benefit-rates 

Will send you and @AnneHurley PDF with comments - more just as a log of edits. 

Was there another page that I needed to look at for this ticket?

**Comment by RLHecht** [comment]
Thanks, @jenniferlee-dsva Did you also look at https://prod.cms.va.gov/2019-va-dic-survivor-dic-rates? Those are the two with this ticket.

**Comment by AnneHurley** [comment]
@jenniferlee-dsva  - Definitely agree the references to "new" spouses is confusing. I will write an email about those proposed changes with a request for clarification. Who are the SMEs for these DIC rates pages?
@Karak888  and @RLHecht  FYI. Will upload edits when I get them.

**Comment by jenniferlee-dsva** [comment]
Replying to question above: https://github.com/department-of-veterans-affairs/va.gov-team/issues/3040#issuecomment-554348613 

@RLHecht and @AnneHurley - reviewed the DIC rates page for spouses and dependents. A couple of major points WRT H1 and H2s. Others more granular and I'm sure I missed a lot as well. 

See attached PDF with comments and changes -- did  not make these in Drupal. Please note some changes are tracking back to the other DIC rates page, to be more parallel/consistent between these two. So will need making parallel changes on the other DIC rates page. I may have missed noting some places, but please do apply the same conceptually to both when needed. Thanks for your patience! 

**PDF with comments - DIC rates for spouses and dependents**
[DIC Survivor DIC Rates-112219-jl.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3917816/DIC.Survivor.DIC.Rates-112219-jl.pdf)

**Comment by AnneHurley** [comment]
[DIC rates page copy edits-ah]
(https://app.zenhub.com/files/133843125/91421a4c-f064-4a5a-afac-12579c844d88/download)
@RLHecht  Here's my rewritten copy for the lower part of the DIC benefit info page. 
Sending email to Angie Smith about removing "survivor" from H1 on that page.

**Comment by RLHecht** [comment]
@jenniferlee-dsva @AnneHurley  and I incorporated all your changes to the survivor DIC rates page. Please review in Drupal and let us know if these are now both ready to send to SMEs. preview-prod.vfs.va.gov/preview?nodeId=1012 and prod.vfs.va.gov/preview?nodeId=1013

**Comment by AnneHurley** [comment]
Sent to SMEs to approve:
preview-prod.vfs.va.gov/preview?nodeId=1012
preview-prod.vfs.va.gov/preview?nodeId=1013

**Comment by Karak888** [comment]
Putting this in "current sprint" column since it is awaiting SME feedback.

**Comment by AnneHurley** [comment]
[Teaser blurbs from DIC to Survivor Pension and Survivor Pension to DIC](https://app.zenhub.com/files/133843125/4607637f-6c1f-4e8e-8c83-5c3dd17d0ada/download)Add DIC teaser text and links to DIC and DIC rates pages on the Survivors Pension page: https://www.va.gov/pension/survivors-pension/
[ ] Vice versa to ^: Add Survivors Pension teaser text and link  on DIC page after it's moved to Disability hub
@RLHecht

**Comment by AnneHurley** [comment]
[teaser blurbs for DIC and Survivor Pension pages - v2](https://app.zenhub.com/files/133843125/a4124c71-b1f9-4870-bf30-5481fba1ca0c/download)
@RLHecht  I decided to keep the link to the Survivors Pension page just the single one since that's how that information is presented. I tightened up the copy for it though.

As for the DIC rates blurbs - @jenniferlee-dsva  - is there a reason we don't include links on that main DIC rates info page to the legacy pages that have the actual current rates? It seems like one of the biggest questions I have as a user.

Here's the page:
https://www.va.gov/disability/dependency-indemnity-compensation/

Right now we have links under More Resources in left nav to what the current rates are for spouses and for children and parents. But couldn't we rewrite that section in the middle of the page and add those links in the body of the page? I'd be looking for it on this page if I were interested in this topic. 

Suggest something like this:

H2: What kind of benefits can I get?
H2: If you qualify, you can get tax-free monetary benefits. The amount you receive depends on the type of survivor you are. 
DIC benefit rates for surviving spouses and children
[[https://www.benefits.va.gov/Compensation/current_rates_dic.asp]]
DIC benefits for surviving parents 
[[https://www.benefits.va.gov/Pension/current_rates_Parents_DIC_pen.asp]]

Would something like that work? We could even keep ^those two links under More Resources in left nav if we wanted to.

**Comment by RLHecht** [comment]
@AnneHurley Per Mikki's comment in the main DIC ticket, we should be including links to the two DIC rates pages on the main DIC page. I missed that from the beginning, so you're right that we should be including info. Also, I am going to copy your above comments to ticket 1727 because this work will finish out that ticket. That way we keep it all in one place. This issue is for the new rates pages.

**Comment by Karak888** [comment]
@RLHecht @AnneHurley is there a reason this is still open?  I thought this had finished but maybe I am mistaken?  Thank

**Comment by RLHecht** [comment]
@Karak888 These two pages are still awaiting SME feedback as far as I know. @AnneHurley have you heard anything from them?

**Comment by AnneHurley** [comment]
@RLHecht @mnorthuis @Karak888  - Checking.

**Comment by AnneHurley** [comment]
@Karak888  @jenniferlee-dsva  @RLHecht  @mnorthuis  - I checked and have not heard back from Angie and team, so I re-sent the email with the Prod links and asking for feedback on the two new DIC pages. I will check back in with them Monday.

**Comment by Karak888** [comment]
@AnneHurley to double check with SMEs (out for holidays).

**Comment by RLHecht** [comment]
@anne Here are the new preview links to send to the SMEs. I am slacking you Word docs with screen shots of both pages for you to look over before we send them to them.

preview-prod.vfs.va.gov/preview?nodeId=1012 (rates for spouses and dependents)
preview-prod.vfs.va.gov/preview?nodeId=1013 (rates for parents)

**Comment by Karak888** [comment]
@AnneHurley  are we still waiting on feedback?  If so, I am moving this to the "current sprint" pipeline and adding awaiting feedback label.

**Comment by AnneHurley** [comment]
@jenniferlee-dsva @RLHecht  @Karak888 

 Am making screenshots of preview-prod.vfs.va.gov/preview?nodeId=1013 (rates for parents) for Angie to send today. The other page has been approved by her.

**Comment by AnneHurley** [comment]
[Survivor Pension Rates-Parents-screenshots for SMEs - 1](https://app.zenhub.com/files/133843125/b46d7ddd-5dac-492b-9486-bf57fdd254b1/download)

**Comment by AnneHurley** [comment]
[Survivor Pension Rates - Both Parents - screenshots for SMEs -2](https://app.zenhub.com/files/133843125/615348ad-be0d-456b-86b0-7ca488a8a0f1/download)

**Comment by AnneHurley** [comment]
[Survivor Pension Rates - 1 parent w other parent or new spouse](https://app.zenhub.com/files/133843125/4d96955f-b952-429d-9b0f-a892f8638657/download)

**Comment by AnneHurley** [comment]
@jenniferlee-dsva  @Karak888  @RLHecht  Sent the above PDFs to Angie and Robert for their edits/updates 1/15

**Comment by AnneHurley** [comment]
[DIC rates - 1 surviving parent]
(https://app.zenhub.com/files/133843125/9b08cbe9-750e-490e-93c1-4502ac1d0318/download)

**Comment by AnneHurley** [comment]
@Karak888  @RLHecht  @jenniferlee-dsva  Screenshots sent to  Angie and Rob.

**Comment by Karak888** [comment]
Awesome job @AnneHurley !!! Thanks @RLHecht per usual.

**Comment by MickinSahni** [comment]
Hey team, safe to say this is no longer awaiting feedback and is complete? cc @AnneHurley @RLHecht

**Comment by AnneHurley** [comment]
@MickinSahni  Unfortunately we heard late last week Angie Smith has some issues still with the Parents rates page; have pinged her about setting up a meeting to go over those changes and hopefully make them quickly. The Surviving Spouse - Children page is ready to go live, but we thought the two should go live at the same time. @jenniferlee-dsva @RLHecht  fyi

**Comment by MickinSahni** [comment]
@AnneHurley if we're still waiting, could this go into Backlog? If it's ready, can we commit to it as part of Sprint 14?

**Comment by AnneHurley** [comment]
@MickinSahni If it's less confusing we could move to Backlog. It's just waiting on a meeting and final-final feedback from the SME - so in theory we could finish it as soon as she's available to meet. I thought we'd have had that info by now.

**Comment by MickinSahni** [comment]
Thanks, @AnneHurley. Pulled it into Backlog and will assign to a sprint when we're ready to go

**Comment by AnneHurley** [comment]
@peggygannon Would you mind taking a look at this page and the tables when you have a little time? Both @RLHecht and I have been working on it and think it would be great to have your fresh eyes. We changed the dollar ranges in all the charts per Angie Smith. 

https://prod.cms.va.gov/2020-va-dic-rates-for-parents

FYSA @jenniferlee-dsva

**Comment by peggygannon** [comment]
@AnneHurley Hello, two recommendations for this page: 
https://prod.cms.va.gov/2020-va-dic-rates-for-parents

--Add "yearly" to the table subheads to make it more clear that the income limits are yearly (as opposed to monthly)
--Update the example in Bullet #1 on submay map. The example in the subway map doesn't match the table above it.

**Comment by AnneHurley** [comment]
@peggygannon  Thanks so much for the close edit. Have added "annual" in all the tables as well as the intro text blurbs. 

Have also revised the first step in the subway map to be clearer now that we have changed the figures that start the columns.

@RLHecht  This could probably use one more review before we send prod links, if you have time.

**Comment by MickinSahni** [comment]
Received updated info for this ticket and therefore pulled out of Backlog and into Sprint 14. 
Waiting for SME feedback to finalize.

**Comment by MickinSahni** [comment]
2020-02-06 PW Standup
1. scrub rates table - Anne
2. then push in Drupal - Randi

**Comment by AnneHurley** [comment]
@RLHecht @peggygannon  this page is ready for your review: https://cms.va.gov/prod/node/1013/edit

What changed:
1. All references to "annual" income are now "yearly" income
2. Peggy had pointed out that in the subway map example we had the wrong starting amount - we'd had $14 from the next income level up, but it should have been $6. I've redone all the math but a) can you please make sure I made the right assumption and picked the correct starting amount  and b) check my math because, you know, math. thanks! After your approval this is ready to go to production.

**Comment by RLHecht** [comment]
@mnorthuis These two DIC pages are ready to go live today. I want to confirm the proper URLs and location are still the ones referenced in ticket #1729.

**Comment by mnorthuis** [comment]
Yes, those are correct. 

/disability/survivor-dic-rates/
/disability/parent-dic-rates/

**Comment by MickinSahni** [comment]
2020-01-07 PW Standup
* Ready to publish (and close this ticket) today!
* Next: redirect tickets - @AnneHurley

**Comment by RLHecht** [comment]
These two rates pages are published! Moving to validate. @mnorthuis @jenniferlee-dsva see screen shots below for IA![Screen Shot 2020-02-07 at 12.15.10 PM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/90274a36-92f3-4304-9b9b-6324f6938f2d)![Screen Shot 2020-02-07 at 12.15.22 PM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/1d3eabee-0bfc-4e02-93b1-944d56c1ecfc)

**Comment by AnneHurley** [comment]
@RLHecht  - great, I will put in the redirect requests now.

**Comment by mnorthuis** [comment]
@RLHecht The DIC rates pages should be after the compensation rates on the hub page and in the left nav.   See issue #1729 for details.

**Comment by RLHecht** [comment]
Sorry, @mnorthuis  I knew I wasn't going to do this right the first time! I moved the order but @AnneHurley can you please provide TWO links and TWO blurbs for the disability hub page for each of these rates pages? They will go under More Resources. Didn't realize it was part of this ticket.
<img width="311" alt="Screen Shot 2020-02-07 at 12 31 14 PM" src="https://user-images.githubusercontent.com/48728214/74051335-be569c00-49a5-11ea-8f31-acfdc1f7ced5.png">

**Comment by mnorthuis** [comment]
These older tickets are hard to follow since the info is all over the place, I think we'll find a better groove as we we start new work!

**Comment by AnneHurley** [comment]
[Disability Hub-DIC teasers]
(https://github.com/department-of-veterans-affairs/va.gov-team/files/4172526/Disability.Hub-DIC.teasers.docx)
@RLHecht  @mnorthuis  Let me know if these will work (they are the same as the ones in the Survivors Pension > More Resources section)

**Comment by RLHecht** [comment]
@peggygannon I added these two links to the disability hub landing page. Could you please review and publish if OK? It's preview-prod.vfs.va.gov/preview?nodeId=68

**Comment by RLHecht** [comment]
FYI I updated the links on the main DIC page to go to these new rates pages with the Node ID.

**Comment by MickinSahni** [comment]
2020-02-10 PW Standup
- Anticipating validating after today's deployment - Randi
- Linked to #5655

**Comment by RLHecht** [comment]
These pages are now live! There are also links on the disability hub page to both rates pages and they are showing in the left nav. Closing ticket. @MickinSahni @AnneHurley @mnorthuis
